config/tracks/spa.ini

```diff
@@ -690,7 +690,7 @@ DESCRIPTION=windows on houses
 MATERIALS=Glass
 KEY_0=ksEmissive
 VALUE_0=205,124,66,0.022
-VALUE_0_OFF=0, 0, 0
+VALUE_0_OFF=205,124,66,0.004

```

Comment:
Do we really need very low lit windows on daylight? Won't remove performance?

Reply:
Although not completely necessary, it certainly adds depth to the pit windows even during daytime at no extra cost to performance, since the shader for CSP room windows are already built while loading the config. The 'Glass' windows are still bright enough for the lights inside to subtly show during daytime.

![20241219052516_1](https://github.com/user-attachments/assets/78358291-a1be-47af-9c4f-560d9c4e7541)